docs/hub/api.md

```diff
@@ -224,4 +230,100 @@ Payload:
 headers = { "authorization" :  "Bearer $token" }
 ```
 
-This is equivalent to `huggingface_hub.whoami()`.
\ No newline at end of file
+This is equivalent to `huggingface_hub.whoami()`.
+
+## Collection API
+
+Use Collections to group repositories from the Hub (Models, Datasets, Spaces and Papers) on a dedicated page.
+
+Find out more about collections [in this guide](./collections.md). Collections can also be managed using the Python client (see [guide](https://huggingface.co/docs/huggingface_hub/main/en/guides/collections)).
+
+### POST /api/collections
+
+Create a new collection on the Hub with a title, a description (optional) and a first item (optional). An item is defined by a type (`model`, `dataset`, `space` or `paper`) and an id (repo_id or paper_id on the Hub).
```

Comment:
it's weird to me that you can create a collection with a first item (but not several items, i assume?) vs. creating a collection then adding an item from a differnt call.

(this comment is about the API design, not really about the documentation)

Reply:
Yes I agree. I assume this mostly comes from the fact that on the Hub we can create a collection from a model page (which means creating it with a single item). 

FYI, what I did in `huggingface_hub` is to not support creating a collection with an item directly. Users have to create collection and then add items to it.

Reply:
ah ok, perfect if you did that in hfh then